3d.js

```diff
@@ -2,195 +2,67 @@ var urlParams = new URLSearchParams(window.location.search);
 var urltoload = urlParams.get("load");
 var baselayer = "hybrid";
 if (urlParams.has("base")) {
-    baselayer = urlParams.get("base"); //maybe sanitise this?? Or allow only a reliable subset.
-    //I think there are too many options in the ESRI base layers and some 3rd party ones don't work.
-    // eg: just hybrid (default), satellite, street, terrain, dark.
+  baselayer = urlParams.get("base"); //maybe sanitise this?? Or allow only a reliable subset.
+  //I think there are too many options in the ESRI base layers and some 3rd party ones don't work.
+  // eg: just hybrid (default), satellite, street, terrain, dark.
 }
```

Comment:
Does this URL parameter still take effect? If not, we may remove this part for all views.

Reply:
Its is not in effect because we move the map style configuration into JSON data

Reply:
Then please remove this part for all views.